src/third_party/ffmpeg.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+import logging
+
+
+def check_ffmpeg_installed():
```

Comment:
is_ffmpeg_installed

Reply:
Also, it seems like twose functions could be merged into some generic code with parameters.